src/Libraries/RevitNodes/Elements/ImportInstance.cs

```diff
@@ -102,7 +102,9 @@ private void InternalSetImportInstance(Autodesk.Revit.DB.ImportInstance ele)
         }
 
         #region Public properties
-
+        /// <summary>
+        /// Gets the specified string from import instance
```

Comment:
Gets file path of sat file that represents the given import instance geometry

@pboyer, Do we need to expose this property? This gives the path of a temp file, so may not be needed for users.